src/components/AnimalDetail/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {useState, useEffect} from "react";
+import './style.css';
+//nezapomínej změnit class na className
+//nechápu, proč mi nefunguje destructure props na foto, nazev, nazevLatinsky, popis, domovina, biotop, potrava, velikost
```

Comment:
už to funguje :))

Reply:
Jinak takovéhle věci se většinou prefixují
TODO nebo FIXME, a pak existují doplňky/editory co ti vypíšou seznam všech TODO komentářů https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree